doc/user/content/sql/copy-from.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "COPY FROM"
+description: "`COPY FROM` copies data into a table using the COPY protocol."
+menu:
+    main:
+        parent: "sql"
+---
+
+`COPY FROM` copies data into a table using the [Postgres COPY protocol](https://www.postgresql.org/docs/current/sql-copy.html).
+
+## Syntax
```

Comment:
Needs documentation for the column list support. The SVG graph was already updated.